### PR TITLE
Create archway-stargaze.json

### PR DESCRIPTION
**Pull request by nodersteam**

### CODE REVIEW

File: _IBC/archway-stargaze.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      "chain_1": {
+        "address": "archway14n8p425hz5l6xxcrq4ml00djkmwezm0m0uunqz"
+      },
+      "chain_2": {
+        "address": "stars14n8p425hz5l6xxcrq4ml00djkmwezm0mwth2py"
+      },
+      "memo": "[NODERS]TEAM Relayer | hermes 1.7.0+e98bd7e83 (https://hermes.informal.systems)",
+      "name": "[NODERS]TEAM",
+      "discord_handle": "nodersseptima"
```
Comment (jlehtimaki) [inline _IBC/archway-stargaze.json:40]:
```suggestion
      "discord": {
        "handle": "nodersseptima",
        "id": "811947913537519647"
      }
```

**Review by jlehtimaki — CHANGES_REQUESTED** [review]
please use the correct json schema

File: _IBC/archway-stargaze.json (hunk 1 above)
Comment (nodersteam) [reply _IBC/archway-stargaze.json]: Fixed

**Review by shahbazn — APPROVED** [review]
lgtm

**Pull request merged**